appengine-java11/helidon-mp-gae-greet/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Application properties. This is the default greeting
+app.greeting=Hello
+
+# Microprofile server properties
+server.port=8080
```

Comment:
```suggestion
server.port=${PORT}
```
set to PORT env var

Reply:
I've observed, if we use any value other than 8080, the app gets deployed but isn't loading on App Engine.